ui/app/templates/components/console/ui-panel.hbs

```diff
@@ -22,14 +22,18 @@
       <Hds::Link::Inline @href={{doc-link "/vault/docs/command/web"}}>HashiCorp Developer site</Hds::Link::Inline>.
     </p>
     <p class="console-ui-panel-intro is-font-mono has-bottom-margin-s">Examples:</p>
-    <p class="console-ui-panel-intro is-font-mono">→ Write secrets to kv v1: write &lt;mount&gt;/my-secret foo=bar</p>
-    <p class="console-ui-panel-intro is-font-mono">→ List kv v1 secret keys: list &lt;mount&gt;/</p>
-    <p class="console-ui-panel-intro is-font-mono">→ Read a kv v1 secret: read &lt;mount&gt;/my-secret</p>
-    <p class="console-ui-panel-intro is-font-mono">→ Mount a kv v2 secret engine: write sys/mounts/&lt;mount&gt; type=kv
-      options=version=2</p>
-    <p class="console-ui-panel-intro is-font-mono">→ Read a kv v2 secret: kv-get &lt;mount&gt;/secret-path</p>
-    <p class="console-ui-panel-intro is-font-mono">→ Read a kv v2 secret's metadata: kv-get &lt;mount&gt;/secret-path
-      -metadata</p>
+    <p class="console-ui-panel-intro is-font-mono">
```

Comment:
I wonder if we should add instructions about closing the window with `ESC` key here, particularly if the close button will sometimes be out of the screen view